### PR TITLE
Ypersky multi clone dashboard

### PR DESCRIPTION
**Pull request by ypersky1980**
This PR implements sending test results to the ES and pushing the results to Performance Dashboard.

### CODE REVIEW

File: tests/e2e/performance/csi_tests/test_pvc_multi_clone_performance.py [hunk 1]
```diff
@@ -3,37 +3,85 @@
...
+from ocs_ci.helpers.helpers import get_full_test_logs_path
 from ocs_ci.ocs import constants, exceptions
 from ocs_ci.ocs.cluster import CephCluster
 from ocs_ci.utility.utils import ocsci_log_path
 from ocs_ci.helpers import performance_lib
 
 log = logging.getLogger(__name__)
 
-SKIP_REASON = "Skipping multiple clone tests until github issue 5198 is addressed"
+# SKIP_REASON = "Skipping multiple clone tests until github issue 5198 is addressed"
```
Comment (ramkiperiy) [inline tests/e2e/performance/csi_tests/test_pvc_multi_clone_performance.py:28]: are we removing this skip?

Comment (ramkiperiy) [reply tests/e2e/performance/csi_tests/test_pvc_multi_clone_performance.py:28]: github https://github.com/red-hat-storage/ocs-ci/issues/5198 issues is due to teardown in test_multi_snapshot.py tests, don't think we are observing this problem in recent runs, correct me if I am wrong here.

Comment (Avilir) [reply tests/e2e/performance/csi_tests/test_pvc_multi_clone_performance.py:28]: +1 for @ramkiperiy , this line should be deleted and not markout

Comment (ypersky1980) [reply tests/e2e/performance/csi_tests/test_pvc_multi_clone_performance.py:28]: yes, I've executed the test locally and it does not leave leftovers. I think the skip should be removed, and if we see any problem in the future - we'll address it then. There is not reason as I see it to skip the test now.

Comment (ypersky1980) [reply tests/e2e/performance/csi_tests/test_pvc_multi_clone_performance.py:28]: @Avilir sure, will delete it. Thanks!

**Comment by keemano** [comment]
LGTM

**Review by pintojoy — APPROVED** [review]
LGTM

**Review by Avilir — APPROVED** [review]
Nice work @ypersky1980

**Review by ramkiperiy — APPROVED** [review]
LGTM

**Pull request merged**